shared/src/proxies/customCaseReport/createCsvCustomCaseReportFileProxy.ts

```diff
@@ -0,0 +1,13 @@
+import { CustomCaseReportCsvRequest } from '@web-api/business/useCases/customCaseReport/createCsvCustomCaseReportFileInteractor';
+import { post } from '../requests';
+
+export const createCsvCustomCaseReportFileInteractor = (
+  applicationContext,
+  data: CustomCaseReportCsvRequest,
+) => {
+  return post({
+    applicationContext,
+    body: data,
+    endpoint: '/async/reports/csv/custom-case-report',
```

Comment:
somewhat nitpicky, but could this be rearranged please?

`/async/reports/custom-case-report/csv`

I'd also consider adding format=csv into a request variable